ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -38,7 +48,37 @@ def start(self) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("Cluster start operation is not implemented.")
+        _, _, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
+        if _rc != 0:
+            self._execute.run_cmd(const.PCS_CLUSTER_START, check_error=False)
+            time.sleep(60)
+            _, _, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
+            if _rc != 0:
+                return {"status": const.STATUSES.FAILED.value, "msg": "Cluster start operation failed"}
+
+        _res = self.node_list()
+        _res = json.loads(_res)
+
+        if _res.get("status") == const.STATUSES.SUCCEEDED.value:
+            _node_list = _res.get("msg")
+            if _node_list is not None:
+                _node_group = [ _node_list[i:i + const.PCS_NODE_START_GROUP_SIZE] for i in range(0, len(_node_list), const.PCS_NODE_START_GROUP_SIZE)]
+
+                for _node_subgroup in _node_group:
+                    for _node_id in _node_subgroup:
+                        _res = self._controllers[const.NODE_CONTROLLER].start(_node_id)
+                        _res = json.loads(_res)
+                        if _res.get("status") == const.STATUSES.FAILED.value:
+                            msg = _res.get("msg")
+                            Log.error(f"Node {_node_id} : {msg}")
+                    time.sleep(30)
+
+                return {"status": const.STATUSES.IN_PROGRESS.value, "msg": "Cluster start operation performed"}
+            else:
+                return {"status": const.STATUSES.FAILED.value, "msg": "Node list is empty"}
```

Comment:
for node list failed msg
msg: 
Cluster start failed. Not able to verify node list.

Reply:
Done

Reply:
Missed one change

`{"status": const.STATUSES.FAILED.value, "msg": "Node list is empty"}`

Reply:
(optional)
Also for any error you can raise
ClusterManagerError

@controller_error_handler will covert error to json

Reply:
Done